packages/js/package.json

```diff
@@ -51,22 +56,20 @@
     "moment-duration-format": "^2.2.2",
     "prop-types": "^15.5.10",
     "react-animate-height": "^2.0.23",
+    "react-aria-live": "^2.0.5",
+    "react-chartjs-2": "^5.2.0",
     "react-helmet": "^6.1.0",
+    "react-hotkeys-hook": "^4.0.5",
     "react-intl": "^2.4.0",
     "react-redux": "^5.0.6",
+    "react-router-dom": "^6.3.0",
     "react-select": "^3.1.0",
     "redux": "^3.7.2",
     "redux-thunk": "^2.2.0",
     "styled-components": "^5.3.6",
     "yoast-components": "^5.24.0",
     "yoastseo": "^1.91.1",
```

Comment:
A lot of changes to this file and new libs that I don't think are relevant to this branch?

Reply:
Those are not newly added packages. It's the lerna/yarn add that fixes the package's order if it's incorrect, and those were not sorted correctly previously. Github doesn't make it clear when it's about a line change, maybe better to use vscode for review, it's more clear. ;)

Reply:
![image](https://github.com/wincher-ab/wordpress-seo/assets/129404933/84d87d6b-f881-4bf4-be10-cfc093941586)